python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -557,14 +676,61 @@ def build_sensor_context(
     )
 
 
+def build_multi_asset_sensor_context(
+    asset_keys: Sequence[AssetKey],
+    instance: Optional[DagsterInstance] = None,
+    cursor: Optional[Mapping[AssetKey, str]] = None,
+    repository_name: Optional[str] = None,
+) -> SensorEvaluationContext:
+    """Builds multi asset sensor execution context using the provided parameters.
+
+    This function can be used to provide a context to the invocation of a multi asset sensor definition. If
+    provided, the dagster instance must be persistent; DagsterInstance.ephemeral() will result in an
+    error.
+
+    Args:
+        asset_keys (Sequence[AssetKey]): The list of asset keys monitored by the sensor
+        instance (Optional[DagsterInstance]): The dagster instance configured to run the sensor.
+        cursor (Optional[Mapping[AssetKey, str]]): A dictionary of cursor values to provide to the evaluation of the sensor.
+        repository_name (Optional[str]): The name of the repository that the sensor belongs to.
+
+    Examples:
+
+        .. code-block:: python
+
+            context = build_multi_asset_sensor_context()
```

Comment:
update this example to pass instance and asset keys since those are required for this to work